src/Cargo.toml

```diff
@@ -21,7 +21,7 @@ path = "main.rs"
 [dependencies]
 uart_16550 = "0.3.0"
 spin = "0.5.2"
-log = { workspace = true }
+log = { workspace = true, features = ["release_max_level_info"] }
```

Comment:
We don't want to disable the logs all the time, otherwise we simply can't log anymore.

What we want to do is to add new features to Miralis that disable the logs based on the config file. The runner has to compute the max log level based on the config, and then pass the appropriate feature to Miralis.

Here is how to disable dependency features from a crate's own features: https://doc.rust-lang.org/cargo/reference/features.html#dependency-features